tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -6020,7 +6020,7 @@ TEST_F(HVACVRFFixture, VRFTest_CondenserCalcTest)
     EXPECT_EQ(VRF(VRFCond).ElecCoolingPower, 0.0);
 
     // make adjustment for heat recovery startup degradation
-    Real64 HREIRFTConst = VRF(VRFCond).HRCAPFTHeatConst;
+    Real64 HREIRFTConst = VRF(VRFCond).HREIRFTHeatConst;
```

Comment:
This is an existing test. Which will keep working anyways since it doesn't have HR curves